sickbeard/postProcessor.py

```diff
@@ -924,6 +924,26 @@ def process(self):
 
                 sql_l.append(cur_ep.get_sql())
 
+                data  = {
+                        'seasons': [
+                            {
+                                'number': epObj.season,
+                                'episodes': [
+                                    {
+                                        'number': epObj.episode
+                                    }
+                                ]
+                            }
+                        ]
+
+                 }
+
+                datas.update(data)
+
+        if sickbeard.USE_TRAKT:
```

Comment:
in the next commit is already changed. i have left USE_TRAKT becouse if you disable trakt notification but TRAKT_SYNC_WATCHLIST is enabled i think that it will update watchlist